recipes-firmware/firmware/firmware-qcom-boot-common.inc

```diff
@@ -0,0 +1,20 @@
+# Install NHLOS boot binaries in DEPLOY_DIR
```

Comment:
Do we expect having multiple versions of boot firmware as part of the same layer? Or is this to be shared with boot firmware recipes for other SoC families?

Reply:
Yes, this is intended to be shared with other SoC families. We need to post the recipe for the SA8775P as well.